scripts/generate_samples.ts

```diff
@@ -0,0 +1,88 @@
+import fs from "fs";
+import { chromium } from "@playwright/test";
+
+// Declare a global augmentation for the Window interface
+declare global {
+  interface Window {
+    map: {
+      loaded: () => boolean;
+    };
+  }
+}
+
+type LocationClip = {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+};
+
+type SampleSpecification = {
+  location: string;
```

Comment:
Document these? location is `z/lon/lat`, for instance. They're all apparently required, are there defaults?

The web page has a bunch of stuff on it, I think clip is intended to select a rectangle from the middle but it might be nice to have a web page without the stuff on it so that clip can be "the whole viewport".

Reply:
Should we just have a separate web page without the controls that we can use as an option and that drop the clip entirely?  That would certainly simplify this config.